Kitodo-ImageManagement/src/test/java/org/kitodo/imagemanagement/ImageManagementIT.java

```diff
@@ -1,140 +1,140 @@
-     */
-    @BeforeClass
-    public static void createTestTiff() throws IOException, InterruptedException, IM4JavaException {
-
-        // create the resources directory if it is not there
-        File resourcesDirectory = new File(RESOURCES);
-        if (!resourcesDirectory.exists()) {
-            resourcesDirectory.mkdirs();
-        }
-
-        // create the test tiff
-        IMOperation operation = new IMOperation();
-        operation.addRawArgs(Arrays.asList("wizard:", "-compress", "None", _00000001_TIF));
-        ConvertCmd convertCmd = new ConvertCmd();
-        if (SystemUtils.IS_OS_WINDOWS) {
-            String parameter = KitodoConfig.getParameter(ParameterImageManagement.SEARCH_PATH);
-            convertCmd.setSearchPath(parameter);
-        }
-        convertCmd.run(operation);
-    }
-
-    @Test
-    public void testGetScaledWebImage() throws IOException, InfoException {
-        assert new File(_00000001_TIF).exists();
-        ImageManagementInterface module = new ImageManagement();
-        Image scaledWebImage = module.getScaledWebImage(new File(_00000001_TIF).toURI(), 0.3);
-        assertEquals((int) Math.round(0.3 * new Info(_00000001_TIF, true).getImageWidth()),
-            scaledWebImage.getWidth(null));
-    }
-
-    @Test
-    public void testCreateDerivative() throws IOException, InfoException {
-        assert new File(_00000001_TIF).exists();
-        File maxDerivative = new File(_00000001_MAX_JPG);
-        ImageManagementInterface module = new ImageManagement();
-        assertTrue(
-            module.createDerivative(new File(_00000001_TIF).toURI(), 1.0, maxDerivative.toURI(), ImageFileFormat.JPEG));
-        assertTrue(maxDerivative.exists());
-        assertEquals(new Info(_00000001_TIF, true).getImageWidth(), new Info(_00000001_MAX_JPG, true).getImageWidth());
-    }
-
-    @Test
-    public void testChangeDpi() throws IOException, InfoException {
-        assert new File(_00000001_TIF).exists();
-        ImageManagementInterface module = new ImageManagement();
-        Image image = module.changeDpi(new File(_00000001_TIF).toURI(), 300);
-        assertEquals(300, 72 * image.getWidth(null) / new Info(_00000001_TIF, true).getImageWidth());
-
-    }
-
-    @Test
-    public void testGetSizedWebImage() throws IOException {
-        assert new File(_00000001_TIF).exists();
-        ImageManagementInterface module = new ImageManagement();
-        Image scaledWebImage = module.getSizedWebImage(new File(_00000001_TIF).toURI(), 150);
-        assertEquals(150, scaledWebImage.getWidth(null));
-    }
-
-    /**
-     * Clean up after tests.
-     */
-    @AfterClass
-    public static void cleanUp() {
-        File tiffFile = new File(_00000001_TIF);
-        if (tiffFile.exists()) {
-            tiffFile.delete();
-        }
-        File jpgMaxFile = new File(_00000001_MAX_JPG);
-        if (jpgMaxFile.exists()) {
-            jpgMaxFile.delete();
-        }
-    }
-}
+///*
```

Comment:
Are you sure that you want to disable this test class?

Reply:
Of course not X(